m4/xbmc_arch.m4

```diff
@@ -36,7 +67,7 @@ case $host in
      AC_SUBST(ARCH_DEFINES, "-DTARGET_POSIX -DTARGET_LINUX -D_LINUX -DTARGET_ANDROID")
      ;;
   *)
-     AC_MSG_ERROR(unsupported host ($host))
+     AC_MSG_ERROR(unsupported host (target $host))
```

Comment:
host = native build machine. could you make it:
AC_MSG_ERROR(unsupported build target: $host)